### ISSUE TITLE
Description and groups fail to be parsed

### ISSUE THREAD

**Issue by zbalkan** [zbalkan/rulevis#7]
# Summary

Reported by @branchnetconsulting

> This rule does not seem to be getting its description or groups parsed successfully.  Could the `&` in the description be possibly related?
> 
> <img width="439" height="853" alt="Image" src="https://github.com/user-attachments/assets/3a1d9bb5-8172-40d3-a6f3-a5a552cebd8c" />
>
> ```xml
>         <rule id="250590" level="14">
>                 <if_sid>61605</if_sid>
>                 <field name="win.eventdata.DestinationPort">^88$</field>
>                 <field name="win.eventdata.Initiated">true</field>
>                 <description>ATT&CK T1208: Suspicious Outbound Kerberos Connection</description>
>                 <info type="text">Detects suspicious outbound network activity via kerberos default port indicating possible lateral movement or first stage PrivEsc via delegation. </info>
>                 <info type="text">Falsepositives: Other browsers. </info>
>                 <info type="text">Sigma UUID: e54979bd-c5f9-4d6c-967b-a04b19ac4c74 </info>
>                 <info type="link">https://github.com/GhostPack/Rubeus8 </info>
>                 <group>attack.lateral_movement,attack.t1208,MITRE</group>
>         </rule>
> ```

**Comment by zbalkan** [comment]
It is indeed related to ampersand handling of ElementTree. I need to escape properly.

**Comment by zbalkan** [comment]
Solved by https://github.com/zbalkan/rulevis/pull/10
